TrkReco/src/MergeKalSeeds_module.cc

```diff
@@ -53,7 +65,18 @@ namespace mu2e {
       if(ksch.isValid()){
         auto const& ksc = *ksch;
         if(debug_ > 2) std::cout << seedcolltag << " Has " << ksc.size() << " KalSeeds" << std::endl;
-        for(size_t ikseed = 0; ikseed <ksc.size(); ++ikseed){ mkseeds->emplace_back(ksch,ikseed); }
+        for(size_t ikseed = 0; ikseed <ksc.size(); ++ikseed){
+          auto const& kseed = ksc[ikseed];
+          if( (!selector_) ) {
+            mkseeds->emplace_back(ksch,ikseed);
+          } else if ( selector_->select(kseed)){
+            if(!selbest_ || mkseeds->size() == 0)
```

Comment:
What's the motivation for having both a ```selbest``` boolean and a selector? Doesn't specifying a selector in the configuration imply that we want to use it?